SegmentEditorEngrave/SegmentEditorEngrave.py

```diff
@@ -88,7 +88,7 @@ def test_Engrave1(self):
     slicer.mrmlScene.AddNode(segmentEditorNode)
     segmentEditorWidget.setMRMLSegmentEditorNode(segmentEditorNode)
     segmentEditorWidget.setSegmentationNode(segmentationNode)
-    segmentEditorWidget.setMasterVolumeNode(masterVolumeNode)
+    segmentEditorWidget.SetSourceVolumeNode(sourceVolumeNode)
```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases.